templates/logging-sidecar-configmap.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if .Values.loggingSidecar.enabled }}
```

Comment:
Can you put a file header here like are in many of the other templates? Having one makes helm behave a little nicer when the whole template is wrapped in a check for enabled, causing the template to return some data even when the feature is disabled. Otherwise we can run into errors that are inconsistent between the feature being enabled and disabled.

Reply:
Thanks for the explanation. Added the header comment to the configmap template.